src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/roi_align.cpp

```diff
@@ -19,44 +19,88 @@ const std::vector<InferenceEngine::Precision> netPRCs = {
     // in fp32 and fp16 precisions.
     // In real AI applications this problem is solved by precision-aware training.
 
-    // InferenceEngine::Precision::FP16,
+    // InferenceEngine::Precision::FP16
```

Comment:
FP 16 test actually fails in this single layer test?

Reply:
yes, they do